README.md

```diff
@@ -575,6 +575,8 @@ These items are relevant *only* on Windows.
 
 - An alternative option to get GNU developer tools (such as GCC) on Windows is [MinGW](http://www.mingw.org/) and its [MSYS](http://www.mingw.org/wiki/msys) package, which provides utilities such as bash, gawk, make and grep. MSYS doesn't have all the features compared to Cygwin. MinGW is particularly useful for creating native Windows ports of Unix tools.
 
+- There is an unofficial package manager for Windows - [Chocolatey](https://chocolatey.org/), which is more similar to the likes of `aptitude` and `yum` found in linux distros. 
```

Comment:
It has got no connection with ```bash``` or other shells. It is a windows-only command line package manager to pull software and install in your box. There are [*nix](https://chocolatey.org/packages?q=tag%3Aunix) and [GNU](https://chocolatey.org/packages?q=tag%3Agnu) based tools available to install via Chocolatey, if you're asking about that. But again, it is not just that. You can pull in a variety of software to be used in Windows via this command line package manager.